src/cascadia/TerminalSettingsModel/GlobalAppSettings.h

```diff
@@ -26,6 +26,9 @@ Author(s):
 #include "NewTabMenuEntry.h"
 #include "RemainingProfilesEntry.h"
 
+bool UseMicaAlt() const;
+bool _useMicaAlt = false;

```

Comment:
Please introduce the setting in MTSMSettings.h instead as a part of the `MTSM_GLOBAL_SETTINGS` macro. This should create a lot of the boiler plate code to add a new setting properly.